apps/explorer/src/app/routes/router-config.tsx

```diff
@@ -75,14 +75,43 @@ const partiesRoutes: Route[] = flags.parties
           },
           {
             path: ':party',
-            element: <PartySingle />,
-            handle: {
-              breadcrumb: (params: Params<string>) => (
-                <Link to={linkTo(Routes.PARTIES, params.party)}>
-                  {truncateMiddle(params.party as string)}
-                </Link>
-              ),
-            },
+            element: <Party />,
+
+            children: [
```

Comment:
This feels really clunky, so I think I have done this incorrectly - if anyone has suggestions for how to improve it, I'm happy to hear them!